site/src/components/TemplateVersionEditor/MissingTemplateVariablesDialog.tsx

```diff
@@ -0,0 +1,138 @@
+import { makeStyles } from "@material-ui/core/styles"
+import Dialog from "@material-ui/core/Dialog"
+import DialogContent from "@material-ui/core/DialogContent"
+import DialogContentText from "@material-ui/core/DialogContentText"
+import DialogTitle from "@material-ui/core/DialogTitle"
+import { DialogProps } from "components/Dialogs/Dialog"
+import { FC, useState } from "react"
+import { FormFields, VerticalForm } from "components/Form/Form"
+import { TemplateVersionVariable, VariableValue } from "api/typesGenerated"
+import DialogActions from "@material-ui/core/DialogActions"
+import Button from "@material-ui/core/Button"
+import { VariableInput } from "pages/CreateTemplatePage/VariableInput"
+import { Loader } from "components/Loader/Loader"
+
+export type MissingTemplateVariablesDialogProps = Omit<
+  DialogProps,
+  "onSubmit"
+> & {
+  onClose: () => void
+  onSubmit: (values: VariableValue[]) => void
+  missingVariables?: TemplateVersionVariable[]
+}
+
+export const MissingTemplateVariablesDialog: FC<
+  MissingTemplateVariablesDialogProps
+> = ({ missingVariables, onSubmit, ...dialogProps }) => {
+  const styles = useStyles()
+  const [variableValues, setVariableValues] = useState<VariableValue[]>([])
+
+  return (
+    <Dialog
+      {...dialogProps}
+      scroll="body"
+      aria-labelledby="update-build-parameters-title"
+      maxWidth="xs"
+      data-testid="dialog"
+    >
+      <DialogTitle
+        id="update-build-parameters-title"
+        classes={{ root: styles.title }}
+      >
+        Template variables
+      </DialogTitle>
+      <DialogContent className={styles.content}>
+        <DialogContentText className={styles.info}>
+          There are a few missing template variable values. Please fill them in.
+        </DialogContentText>
+        <VerticalForm
+          className={styles.form}
+          id="updateVariables"
+          onSubmit={(e) => {
+            e.preventDefault()
+            onSubmit(variableValues)
+          }}
+        >
+          {missingVariables ? (
+            <FormFields>
+              {missingVariables.map((variable, index) => {
+                return (
+                  <VariableInput
```

Comment:
It looks like Chrome populates inputs with wrong data. Can we disable autocompletion here?

(`admin@coder.com` is a saved password for Coder)
<img width="517" alt="Screenshot 2023-04-13 at 09 10 11" src="https://user-images.githubusercontent.com/14044910/231682303-e2b9c259-369f-43bb-a982-d2b158faa9d7.png">

<img width="423" alt="Screenshot 2023-04-13 at 09 10 31" src="https://user-images.githubusercontent.com/14044910/231682415-6fcc258a-023c-43fc-8b1a-16f7ee01b941.png">

Reply:
Sure!

Reply:
Looks good now